packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.js

```diff
@@ -36,14 +36,19 @@ const DropdownItem = ({ className, children, isHovered, component: Component, is
   } else if (Component === 'button') {
     additionalProps.disabled = isDisabled;
   }
+
   return (
     <li>
-      <Component
-        {...additionalProps}
-        className={css(isDisabled && styles.modifiers.disabled, isHovered && styles.modifiers.hover, className)}
-      >
-        {children}
-      </Component>
+      {React.isValidElement(children) ? (
+        children
```

Comment:
I think we want to clone the children her an apply the styles.  You can see an example in the NavItem component.

Reply:
I had this implemented initially (see Joachim's comment thread). Would it be better to enforce the Dropdown styling or let the user define their own styles because these are custom elements that are embedded into the DropdownItem?

Reply:
Ah, i didn't see any patternfly classes copied on the children, just className. If there are patternfly styles then we can clone and apply them